### DISCUSSION TITLE
zie/ziewin/3.0/ja/doc/qb/abt

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#74] [Q&A]
# zie/ziewin/3.0/ja/doc/qb/abt

本書は、 HCL Z and I Emulator for Windows のインストール、構成、および始動の方法を説明しています。 Z and I Emulator for Windows をインストールして作動状態にし、各種の作業を始めた後で、 補足情報が必要なときは いつでもオンライン・ヘルプを利用することができます。オンライン・ヘルプ、 Z and I Emulator for ...

https://help.hcl-software.com/zie/ziewin/3.0/ja/doc/qb/abt.html

**Comment by KunioMatsuo** [comment]
"Windows® refers to Microsoft® Windows® 7, Windows® 8/8.1, Windows® 10, Windows® Server 2008, and Windows® Server 2012" is ancient.  This should be updated.
